nncf/common/graph/graph.py

```diff
@@ -71,6 +104,49 @@ class NNCFGraphNodeType:
     OUTPUT_NODE = MODEL_OUTPUT_OP_NAME
 
 
+class NNCFGraphEdge:
+    """A structure describing an edge in NNCFGraph. Since nodes of the NNCFGraph are operations
```

Comment:
Please, format the docstring according to the Style Guide.

Reply:
Done